senz-web/backend/.env

```diff
@@ -0,0 +1,3 @@
+ENV="test"
+EMAIL_PASSWORD = "adi9891304474"
```

Comment:
**Remove** the .env file from the PR and reset your password for security reasons.